RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -600,7 +600,20 @@ private IMenuItem GetToolsParentMenu()
                 KernelInstance.Get<ToDoExplorerCommandMenuItem>()
             };
 
-            return new ToolsParentMenu(items);
+            var itemsSCDisabled = new IMenuItem[]
+            {
+                KernelInstance.Get<RegexAssistantCommandMenuItem>(),
+                KernelInstance.Get<ToDoExplorerCommandMenuItem>()
+            };
+
+            if (KernelInstance.Get<IGeneralConfigService>().LoadConfiguration().UserSettings.GeneralSettings.SourceControlEnabled == false)
+            {
+                return new ToolsParentMenu(itemsSCDisabled);
+            }
+            else
+            {
+                return new ToolsParentMenu(items);
+            }
```

Comment:
Change items to be declared as a List. Then just check to see if we should add the source control feature. 

if (sourceControlEnabled)
{
    items.Add(SourceControlMenuItem);
}

return new ToolsParentMenu(items.ToArray());